UnitTests/UnitTests.UWP/Geometry/Test_RegexFactory.cs

```diff
@@ -0,0 +1,123 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Media.Geometry.Core;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.UWP.Geometry
+{
+    [TestClass]
+    public class Test_RegexFactory
```

Comment:
Should we test some negative cases as well to ensure that the parser handles failure gracefully? (And also doesn't match things it's not supposed to.)

Reply:
Added negative test cases for regexes.